.github/workflows/tests.yml

```diff
@@ -65,8 +67,14 @@ jobs:
         id: install-tardis
         # shell: bash -l {0}
         run: |
-          pip install git+https://github.com/tardis-sn/tardis.git
-          
+            if [ "${{ github.event.action }}" == "Tardis-Dispatch" ]; then
+              echo "Triggered by repository dispatch, installing the latest version of TARDIS."
+              pip install git+https://github.com/${{github.repository_owner}}/tardis.git@${{ github.event.client_payload.commit_sha }}
```

Comment:
sorry if I dont understand this completely,the commit sha, is this intended to be sent by the pull request? will this work as a part of the release workflow? what about workflow dispatch?

Reply:
After giving more thought to this, i believe there is no need for the if else condtions, because it will automatically fetch the latest commit in both cases.